pkg/verification/service/verification_service_test.go

```diff
@@ -134,43 +134,17 @@ func (s *TestVerificationServiceSuite) TestInitVerification() {
 
 	gock.Observe(obs)
 
-	userSignup := &toolchainv1alpha1.UserSignup{
-		TypeMeta: metav1.TypeMeta{},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "123",
-			Namespace: configuration.Namespace(),
-			Labels: map[string]string{
-				toolchainv1alpha1.UserSignupUserPhoneHashLabelKey: "+1NUMBER",
-			},
-		},
-		Spec: toolchainv1alpha1.UserSignupSpec{
-			IdentityClaims: toolchainv1alpha1.IdentityClaimsEmbedded{
-				PreferredUsername: "sbryzak@redhat.com",
-			},
-		},
-	}
+	userSignup := testusersignup.NewUserSignup(
+		testusersignup.WithName("johny"),
+		testusersignup.WithLabel(toolchainv1alpha1.UserSignupUserPhoneHashLabelKey, "+1NUMBER"),
+		testusersignup.VerificationRequired(0))
```

Comment:
does this have the same effect as :
```
states.SetVerificationRequired(userSignup, true)
```
?
It looks like the latter sets the `UserSignupStateVerificationRequired` in the `UserSignup.Spec.States` and it doesn't update the conditions .

Reply:
minor and not related to this PR - also I found it a bit confusing at first to see the `0` value as parameter `testusersignup.VerificationRequired(0)`

Reply:
Yeah, it's weird - I've created a PR to align the names a bit https://github.com/codeready-toolchain/toolchain-common/pull/445

>  It looks like the latter sets the UserSignupStateVerificationRequired in the UserSignup.Spec.States and it doesn't update the conditions

you are right, the `testusersignup.VerificationRequired` updates also the conditions which wasn't done before, technically speaking, the UserSignup should look like that, so it's closer to the real scenario now.